src/FirebirdSql.EntityFrameworkCore.Firebird/Migrations/FbMigrationsSqlGenerator.cs

```diff
@@ -201,6 +201,10 @@ protected override void Generate(CreateIndexOperation operation, IModel model, M
 		{
 			builder.Append("UNIQUE ");
 		}
+		if (operation.IsDescending?.Length > 0 && operation.IsDescending[0])
```

Comment:
Probably worth throwing an exception when some columns are _asc_ and some _desc_.

Reply:
Good idea! Always better to be explicit.